### ISSUE TITLE
[Bug Report]: Problem matcher cannot return the correct file path (VSC-927)

### ISSUE THREAD

**Issue by Pickaxe-Hit** [espressif/vscode-esp-idf-extension#748]
**Pre Bug Report Checklist**
Before reporting any bug please make sure of these points.

- [x] Make sure you have searched for existing bugs and features request before you post an issue.
- [x] This is a bug report for the ESP-IDF Visual Studio Code extension and **not** an ESP-IDF bug report.
- [x] I've read the docs and found no information that could have helped solving the issue.

**Describe the bug**
Problem matcher cannot return the correct file path.

**To Reproduce**
Use esp-idf(master), idf extension 1.4.0.
1.create an esp-idf project
2.type anything which can cause error. such as 
```c
#include <stdio.h>

void app_main(void)
{
test();
}
```
3.build the project
4.see error

**Expected behavior**
problem matcher can work correctly

**Screenshots**
![image](https://user-images.githubusercontent.com/95094338/175561496-415c2630-f391-426a-add7-9dd71fdef21c.png)
when i clicked the problem:
![image](https://user-images.githubusercontent.com/95094338/175561670-738d6346-e31d-479b-886e-4fe96ce5986d.png)

**Environment (please complete the following information):**

- OS Name & Version: Windows build 25140
- VSCode Version: 1.68.1
- ESP-IDF Version: master(v5.0)
- Python Version: 3.8.7

> **NOTE**: You can use the `ESP-IDF: Doctor command` to generate a report of your configuration in your clipboard which you can copy here.

**Output**
```
---------------------------------------------- ESP-IDF Extension for Visual Studio Code report ---------------------------------------------
OS win32 x64 10.0.25140 
System environment variable IDF_PYTHON_ENV_PATH 
 undefined 
System environment variable PATH 
 D:\Program Files\Microsoft\jdk-17.0.1.12-hotspot\bin;C:\Program Files (x86)\Common Files\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\WINDOWS\System32\OpenSSH\;D:\Program Files\Git\cmd;D:\Program Files\platform-tools;D:\Program Files\nodejs\;C:\Program Files\Microsoft VS Code\bin;D:\Users\PickaxeHit\AppData\Local\Programs\Python\Python310\Scripts\;D:\Users\PickaxeHit\AppData\Local\Programs\Python\Python310\;D:\Users\PickaxeHit\AppData\Local\Microsoft\WindowsApps; 
System environment variable PYTHON 
 undefined 
Visual Studio Code version 1.68.1 
Visual Studio Code language zh-cn 
Visual Studio Code shell C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe 
ESP-IDF Extension version 1.3.0 
---------------------------------------------------- Extension configuration settings ------------------------------------------------------
ESP-ADF Path (idf.espAdfPath) ${env:ADF_PATH}
ESP-IDF Path (idf.espIdfPath) D:\Users\PickaxeHit\esp\esp-idf
ESP-MDF Path (idf.espMdfPath) ${env:MDF_PATH}
Custom extra paths (idf.customExtraPaths) D:\Users\PickaxeHit\.espressif\tools\xtensa-esp-elf-gdb\11.2_20220529\xtensa-esp-elf-gdb\bin;D:\Users\PickaxeHit\.espressif\tools\riscv32-esp-elf-gdb\11.2_20220529\riscv32-esp-elf-gdb\bin;D:\Users\PickaxeHit\.espressif\tools\xtensa-esp32-elf\esp-2022r1-RC1-11.2.0\xtensa-esp32-elf\bin;D:\Users\PickaxeHit\.espressif\tools\xtensa-esp32s2-elf\esp-2022r1-RC1-11.2.0\xtensa-esp32s2-elf\bin;D:\Users\PickaxeHit\.espressif\tools\xtensa-esp32s3-elf\esp-2022r1-RC1-11.2.0\xtensa-esp32s3-elf\bin;D:\Users\PickaxeHit\.espressif\tools\riscv32-esp-elf\esp-2022r1-RC1-11.2.0\riscv32-esp-elf\bin;D:\Users\PickaxeHit\.espressif\tools\esp32ulp-elf\2.28.51-esp-20191205\esp32ulp-elf-binutils\bin;D:\Users\PickaxeHit\.espressif\tools\esp32s2ulp-elf\2.28.51-esp-20191205\esp32s2ulp-elf-binutils\bin;D:\Users\PickaxeHit\.espressif\tools\cmake\3.23.1\bin;D:\Users\PickaxeHit\.espressif\tools\openocd-esp32\v0.11.0-esp32-20220411\openocd-esp32\bin;D:\Users\PickaxeHit\.espressif\tools\ninja\1.10.2;D:\Users\PickaxeHit\.espressif\tools\idf-exe\1.0.3;D:\Users\PickaxeHit\.espressif\tools\ccache\4.3\ccache-4.3-windows-64;D:\Users\PickaxeHit\.espressif\tools\dfu-util\0.9\dfu-util-0.9-win64
Custom extra vars (idf.customExtraVars) {"OPENOCD_SCRIPTS":"D:\\Users\\PickaxeHit\\.espressif\\tools\\openocd-esp32\\v0.11.0-esp32-20220411/openocd-esp32/share/openocd/scripts","IDF_CCACHE_ENABLE":"1"}
Virtual env Python Path (idf.pythonBinPath) D:\Users\PickaxeHit\.espressif\python_env\idf5.0_py3.8_env\Scripts\python.exe
Serial port (idf.port) COM1
OpenOCD Configs (idf.openOcdConfigs) board/esp32s3-builtin.cfg
ESP-IDF Tools Path (idf.toolsPath) D:\Users\PickaxeHit\.espressif
Git Path (idf.gitPath) D:\Users\PickaxeHit\.espressif\tools\idf-git\2.30.1\cmd\git.exe
-------------------------------------------------------- Configurations access -------------------------------------------------------------
Access to ESP-ADF Path (idf.espAdfPath) false
Access to ESP-IDF Path (idf.espIdfPath) true
Access to ESP-MDF Path (idf.espMdfPath) false
Access to ESP-IDF Custom extra paths
Access to D:\Users\PickaxeHit\.espressif\tools\xtensa-esp-elf-gdb\11.2_20220529\xtensa-esp-elf-gdb\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\riscv32-esp-elf-gdb\11.2_20220529\riscv32-esp-elf-gdb\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\xtensa-esp32-elf\esp-2022r1-RC1-11.2.0\xtensa-esp32-elf\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\xtensa-esp32s2-elf\esp-2022r1-RC1-11.2.0\xtensa-esp32s2-elf\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\xtensa-esp32s3-elf\esp-2022r1-RC1-11.2.0\xtensa-esp32s3-elf\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\riscv32-esp-elf\esp-2022r1-RC1-11.2.0\riscv32-esp-elf\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\esp32ulp-elf\2.28.51-esp-20191205\esp32ulp-elf-binutils\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\esp32s2ulp-elf\2.28.51-esp-20191205\esp32s2ulp-elf-binutils\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\cmake\3.23.1\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\openocd-esp32\v0.11.0-esp32-20220411\openocd-esp32\bin: true
Access to D:\Users\PickaxeHit\.espressif\tools\ninja\1.10.2: true
Access to D:\Users\PickaxeHit\.espressif\tools\idf-exe\1.0.3: true
Access to D:\Users\PickaxeHit\.espressif\tools\ccache\4.3\ccache-4.3-windows-64: true
Access to D:\Users\PickaxeHit\.espressif\tools\dfu-util\0.9\dfu-util-0.9-win64: true
Access to Virtual env Python Path (idf.pythonBinPath) true
Access to CMake in environment PATH undefined
Access to Ninja in environment PATH undefined
Access to ESP-IDF Tools Path (idf.toolsPath) true
----------------------------------------------------------- Executables Versions -----------------------------------------------------------
Git version 2.30.1.windows.1
ESP-IDF version 5.0
Python version 3.8.7
Python's pip version 20.3.3
-------------------------------------------------- Python packages in idf.pythonBinPath ----------------------------------------------------
bitstring version: 3.1.9
certifi version: 2022.6.15
cffi version: 1.15.0
charset-normalizer version: 2.0.12
click version: 8.1.3
colorama version: 0.4.5
construct version: 2.10.68
contextlib2 version: 21.6.0
cryptography version: 36.0.2
ecdsa version: 0.17.0
esp-coredump version: 1.2
esptool version: 4.1
freertos-gdb version: 1.0.1
future version: 0.18.2
gcovr version: 5.1
idf-component-manager version: 1.1.3
idna version: 3.3
Jinja2 version: 3.1.2
kconfiglib version: 14.1.0
lxml version: 4.9.0
MarkupSafe version: 2.1.1
pip version: 20.3.3
psutil version: 5.9.1
pycparser version: 2.21
pyelftools version: 0.27
pygdbmi version: 0.9.0.2
Pygments version: 2.12.0
pyparsing version: 3.0.9
pyserial version: 3.5
pywin32 version: 304
PyYAML version: 6.0
reedsolo version: 1.5.4
requests version: 2.28.0
requests-toolbelt version: 0.9.1
schema version: 0.7.5
setuptools version: 62.6.0
six version: 1.16.0
tqdm version: 4.64.0
urllib3 version: 1.26.9
websocket-client version: 1.3.3
wheel version: 0.36.2
windows-curses version: 2.3.0
xmlrunner version: 1.7.7
---------------------------------------------------- Check ESP-IDF python requirements.txt -------------------------------------------------
Check ESP-IDF Python packages Error
---------------------------------------------------- Check extension requirements.txt ------------------------------------------------------
Check Extension Python packages Python requirements are satisfied.
---------------------------------------------------- Check ESP-IDF debug adapter requirements.txt ------------------------------------------
Check Debug AdapterPython packages Python requirements are satisfied.
----------------------------------------------------------- Latest error -----------------------------------------------------------------
Latest error at ENOENT: no such file or directory, open 'd:\Users\PickaxeHit\Documents\sample_project\.vscode\launch.json'
--------------------------------------------------------------------------------------------------------------------------------------------
```

In Visual Studio Code select menu `View` -> `Output` -> ESP-IDF, ESP-IDF Debug Adapter, Heap Trace, OpenOCD and SDK Configuration Editor.

> **NOTE:** Use `idf.openOcdDebugLevel` configuration setting to 3 or more to show debug logging in OpenOCD server output.

> **NOTE:** Use `logLevel` in your <project-directory>/.vscode/launch.json to 3 or more to show more debug adapter output.

Please share the output of the channel related to your issue. For example ESP-IDF is used mostly for the extension setup process output.

**Logs**
Please share the log file which can be obtained from:
```
{"message":"Open On-Chip Debugger  v0.11.0-esp32-20220411 (2022-04-11-08:48)\r\nLicensed under GNU GPL v2\r\nFor bug reports, read\r\n\thttp://openocd.org/doc/doxygen/bugs.html\r\n","stack":"Error: Open On-Chip Debugger  v0.11.0-esp32-20220411 (2022-04-11-08:48)\r\nLicensed under GNU GPL v2\r\nFor bug reports, read\r\n\thttp://openocd.org/doc/doxygen/bugs.html\r\n\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:7304\n\tat ChildProcess.exithandler (node:child_process:397:7)\n\tat ChildProcess.emit (node:events:390:28)\n\tat maybeClose (node:internal/child_process:1064:16)\n\tat Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)","level":"error","timestamp":"2022-06-24T14:53:38.212Z"}
{"user":true,"message":"Something went wrong while trying to build the project","stack":"Error: Task ESP-IDF Build exited with code 1\n\tat Function.<anonymous> (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:32100)\n\tat Generator.next (<anonymous>)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:31158\n\tat new Promise (<anonymous>)\n\tat o (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:30903)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:31918\n\tat g.invoke (c:\\Program Files\\Microsoft VS Code\\resources\\app\\out\\vs\\workbench\\api\\node\\extensionHostProcess.js:58:145)\n\tat b.deliver (c:\\Program Files\\Microsoft VS Code\\resources\\app\\out\\vs\\workbench\\api\\node\\extensionHostProcess.js:58:2265)\n\tat v.fire (c:\\Program Files\\Microsoft VS Code\\resources\\app\\out\\vs\\workbench\\api\\node\\extensionHostProcess.js:58:1843)\n\tat f.$onDidEndTaskProcess (c:\\Program Files\\Microsoft VS Code\\resources\\app\\out\\vs\\workbench\\api\\node\\extensionHostProcess.js:87:35582)\n\tat processTicksAndRejections (node:internal/process/task_queues:96:5)","level":"error","timestamp":"2022-06-24T14:53:41.180Z"}
{"message":"Cannot access filePath: ${env:ADF_PATH}","stack":"Error: ENOENT: no such file or directory, access '${env:ADF_PATH}'\n\tat Object.accessSync (node:fs:250:3)\n\tat Object.e.accessSync (node:electron/js2c/asar_bundle:5:8277)\n\tat I (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:4561)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:247561\n\tat Generator.next (<anonymous>)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:242209\n\tat new Promise (<anonymous>)\n\tat l (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:241954)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:247384\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:248464\n\tat Generator.next (<anonymous>)\n\tat s (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:245882)\n\tat processTicksAndRejections (node:internal/process/task_queues:96:5)","level":"error","timestamp":"2022-06-24T14:56:54.165Z"}
{"message":"Cannot access filePath: ${env:MDF_PATH}","stack":"Error: ENOENT: no such file or directory, access '${env:MDF_PATH}'\n\tat Object.accessSync (node:fs:250:3)\n\tat Object.e.accessSync (node:electron/js2c/asar_bundle:5:8277)\n\tat I (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:4561)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:247757\n\tat Generator.next (<anonymous>)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:242209\n\tat new Promise (<anonymous>)\n\tat l (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:241954)\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:247384\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:248464\n\tat Generator.next (<anonymous>)\n\tat s (d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:8:245882)\n\tat processTicksAndRejections (node:internal/process/task_queues:96:5)","level":"error","timestamp":"2022-06-24T14:56:54.166Z"}
{"message":"Open On-Chip Debugger  v0.11.0-esp32-20220411 (2022-04-11-08:48)\r\nLicensed under GNU GPL v2\r\nFor bug reports, read\r\n\thttp://openocd.org/doc/doxygen/bugs.html\r\n","stack":"Error: Open On-Chip Debugger  v0.11.0-esp32-20220411 (2022-04-11-08:48)\r\nLicensed under GNU GPL v2\r\nFor bug reports, read\r\n\thttp://openocd.org/doc/doxygen/bugs.html\r\n\n\tat d:\\Users\\PickaxeHit\\.vscode\\extensions\\espressif.esp-idf-extension-1.3.0\\dist\\extension.js:1:7304\n\tat ChildProcess.exithandler (node:child_process:397:7)\n\tat ChildProcess.emit (node:events:390:28)\n\tat maybeClose (node:internal/child_process:1064:16)\n\tat Process.ChildProcess._handle.onexit (node:internal/child_process:301:5)","level":"error","timestamp":"2022-06-24T14:57:00.473Z"}
{"message":"ENOENT: no such file or directory, open 'd:\\Users\\PickaxeHit\\Documents\\sample_project\\.vscode\\launch.json'","stack":"Error: ENOENT: no such file or directory, open 'd:\\Users\\PickaxeHit\\Documents\\sample_project\\.vscode\\launch.json'","level":"error","timestamp":"2022-06-24T14:57:02.587Z"}
{"user":true,"level":"warn","message":"Extension configuration report has been copied to clipboard with errors","timestamp":"2022-06-24T14:57:02.588Z"}
```

- Windows: `%USERPROFILE%\.vscode\extensions\espressif.esp-idf-extension-VERSION\esp_idf_vsc_ext.log`
- Linux & MacOSX: `$HOME/.vscode/extensions/espressif.esp-idf-extension-VERSION/esp_idf_vsc_ext.log`

**Extension Configuration settings**

```json
{
    "workbench.startupEditor": "none",
    "idf.gitPath": "D:\\Users\\PickaxeHit\\.espressif\\tools\\idf-git\\2.30.1\\cmd\\git.exe",
    "idf.espIdfPathWin": "D:\\Users\\PickaxeHit\\esp\\esp-idf",
    "idf.pythonBinPathWin": "D:\\Users\\PickaxeHit\\.espressif\\python_env\\idf5.0_py3.8_env\\Scripts\\python.exe",
    "idf.toolsPathWin": "D:\\Users\\PickaxeHit\\.espressif",
    "idf.customExtraPaths": "D:\\Users\\PickaxeHit\\.espressif\\tools\\xtensa-esp-elf-gdb\\11.2_20220529\\xtensa-esp-elf-gdb\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\riscv32-esp-elf-gdb\\11.2_20220529\\riscv32-esp-elf-gdb\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\xtensa-esp32-elf\\esp-2022r1-RC1-11.2.0\\xtensa-esp32-elf\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\xtensa-esp32s2-elf\\esp-2022r1-RC1-11.2.0\\xtensa-esp32s2-elf\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\xtensa-esp32s3-elf\\esp-2022r1-RC1-11.2.0\\xtensa-esp32s3-elf\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\riscv32-esp-elf\\esp-2022r1-RC1-11.2.0\\riscv32-esp-elf\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\esp32ulp-elf\\2.28.51-esp-20191205\\esp32ulp-elf-binutils\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\esp32s2ulp-elf\\2.28.51-esp-20191205\\esp32s2ulp-elf-binutils\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\cmake\\3.23.1\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\openocd-esp32\\v0.11.0-esp32-20220411\\openocd-esp32\\bin;D:\\Users\\PickaxeHit\\.espressif\\tools\\ninja\\1.10.2;D:\\Users\\PickaxeHit\\.espressif\\tools\\idf-exe\\1.0.3;D:\\Users\\PickaxeHit\\.espressif\\tools\\ccache\\4.3\\ccache-4.3-windows-64;D:\\Users\\PickaxeHit\\.espressif\\tools\\dfu-util\\0.9\\dfu-util-0.9-win64",
    "idf.customExtraVars": "{\"OPENOCD_SCRIPTS\":\"D:\\\\Users\\\\PickaxeHit\\\\.espressif\\\\tools\\\\openocd-esp32\\\\v0.11.0-esp32-20220411/openocd-esp32/share/openocd/scripts\",\"IDF_CCACHE_ENABLE\":\"1\"}",
    "explorer.confirmDelete": false
}
```

Open Command Palette (F1 or View Menu -> Command Palette) and type `Preferences: Open Settings (JSON)` or use `{CURRENT_PROJECT}/.vscode/settings.json` to get the following extension configuration settings value:

- `idf.espIdfPath` (MacOS or Linux) or `idf.espIdfPathWin` (Windows) also called `Path to locate ESP-IDF framework (IDF_PATH)` in `Preferences: Open Settings (UI)`.
- `idf.pythonBinPath` (MacOS or Linux) or `idf.pythonBinPathWin` (Windows) also called `Python absolute binary path used to execute ESP-IDF Python Scripts` in `Preferences: Open Settings (UI)`.
- `idf.openOcdConfigs` also called `List of configuration files inside OpenOCD Scripts directory` in `Preferences: Open Settings (UI)`.
- `idf.customExtraPaths` also called `Paths to be appended to PATH` in `Preferences: Open Settings (UI)`.
- `idf.customExtraVars` also called `Variables to be added to system environment variables` in `Preferences: Open Settings (UI)`.
- `idf.adapterTargetName` also called `Target name for ESP-IDF Debug Adapter` in `Preferences: Open Settings (UI)`.
- `idf.customAdapterTargetName` is used when `idf.adapterTargetName` is set to `custom`.

**Additional context**

change these (in package.json)
```json
    "problemMatchers": [
      {
        "name": "idfRelative",
        "owner": "cpp",
        "fileLocation": [
          "relative",
          "${workspaceFolder}"
        ],
        "pattern": {
          "regexp": "^\\.\\.(.*):(\\d+):(\\d+):\\s+(warning|error):\\s+(.*)$",
          "file": 1,
          "line": 2,
          "column": 3,
          "severity": 4,
          "message": 5
        }
      },
```
into
```json
    "problemMatchers": [
      {
        "name": "idfRelative",
        "owner": "cpp",
        "fileLocation": [
          "absolute"
        ],
        "pattern": {
          "regexp": "^(.*?):(\\d+):(\\d*):?\\s+(?:fatal\\s+)?(warning|error):\\s+(.*)$",
          "file": 1,
          "line": 2,
          "column": 3,
          "severity": 4,
          "message": 5
        }
      },
```
can solve the problem(equals to use "gcc" problem matcher in cpptools)

**Comment by Pickaxe-Hit** [comment]
![image](https://user-images.githubusercontent.com/95094338/175563992-fb86d273-b92c-40ad-a738-a0442de595e1.png)
the correct one by changing package.json

**Comment by brianignacio5** [comment]
Thanks for the feedback! Added a PR to fix this in the extension with #749
